package.json

```diff
@@ -15,29 +17,22 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@discordjs/builders": "^0.11.0",
-    "@discordjs/rest": "^0.2.0-canary.0",
-    "discord-api-types": "^0.26.1",
-    "discord.js": "^13.6.0",
-    "discord.js-docs": "github:BenjammingKirby/discord.js-docs#typescriptRewrite",
-    "dotenv": "^12.0.3",
-    "fast-xml-parser": "^4.0.1",
-    "flexsearch": "^0.7.21",
-    "glob": "^7.2.0",
-    "node-fetch": "^2.6.6",
-    "pm2": "^5.1.2"
```

Comment:
Why was pm2 removed?

Reply:
Because it was installed but there is no reference to it in the code, nor is it referenced by any other dependency neither top level nor sublevel. In other words, I found no indication that it was being used.